app/controllers/CheckDetailsController.scala

```diff
@@ -75,11 +75,14 @@ class CheckDetailsController @Inject()(
             } yield (pdvData, idData) match {
 
               case (pdvData, Left(idData)) =>
+                val sessionWithNINO = request.session + ("nino" -> pdvData.getNino)
```

Comment:
I think this line is duplicate. Is it good to move line no 76 and remove duplicate entries.

Reply:
I don't think it's a duplicate. There is a line for each case?

Reply:
Sorry @rambabu-posa  - Did not see this in time. We can do this in a future refactor.